org.kde.kasts.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.kasts",
+    "rename-icon": "kasts",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kasts",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
```

Comment:
We also need permission to use pulseaudio and the session bus for MPRIS

Reply:
:man_facepalming: